contracts/KyberPoolMaster.sol

```diff
@@ -450,22 +440,31 @@ contract KyberPoolMaster is Ownable {
 
     /**
      * @dev  Claims rewards for a given group of epochs in all feeHandlers, distribute fees and its share to poolMaster
-     * @param _epochGroup An array of epochs for which rewards are being claimed
+     * @param _epochGroup An array of epochs for which rewards are being claimed. Asc order and uniqueness is required.
```

Comment:
I think we should update the api docs to reflect that the method expects epochs to be in ascending order and to be unique